src/omr_leadsheet/pipeline/_paths.py

```
@@ -0,0 +1,71 @@
+            f"Tried: {', '.join(candidates)}."
+        )
+    return parent / candidates[0]
+
+
+__all__ = [
+    "SONGS_DIR_CANDIDATES",
+    "MXL_DIR_CANDIDATES",
+    "LYRICS_DIR_CANDIDATES",
+    "LEADSHEETS_DIR_CANDIDATES",
+    "find_subdir",
+]
```

Ruff (0.15.12)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)